netbox_librenms_plugin/templates/netbox_librenms_plugin/_cable_sync_content.html

```
@@ -3,16 +3,23 @@
 
 <!-- Cable Sync Table -->
 {% if cable_sync.table %}
+{# Migrated donors must not sync: hiding only the button leaves the POST form live (Enter in a filter still submits), so drop the form in migrated mode. #}
+{% if migrated_to_marker %}
+<div>
+{% else %}
 <form method="post" action="{% url 'plugins:netbox_librenms_plugin:sync_device_cables' cable_sync.object.pk %}">
```

HTMLHint (1.9.2)

[error] Tag must be paired, missing: [ </form> ], start tag match failed [ <form method="post" action="{% url 'plugins:netbox_librenms_plugin:sync_device_cables' cable_sync.object.pk %}"> ] on line 10.

(tag-pair)